src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ShopService {
+    public Storage process(List<FruitTransaction> transactions);
```

Comment:
The method `process` in the interface `ShopService` should not have `public` access modifier as all interface methods are public by default. It's redundant and should be removed to follow Java best practices.